step.yml

```diff
@@ -52,6 +52,12 @@ inputs:
       summary: ""
       description: "Project ARNs can be obtained using the AWS CLI 'devicefarm list-projects' command."
       is_required: true
+  - billing_method: ""
+    opts:
+      title: "Billing Method"
+      summary: ""
+      description: "use METERED or UNMETERED"
```

Comment:
Can you make this a little more descriptive? How about `"Specifies the billing method for a test run. Use \`METERED\` for free tier and pay-as-you-go billing, and \`UNMETERED\` to use pre-paid device slots. See [How to purchase device slots](https://docs.aws.amazon.com/devicefarm/latest/developerguide/how-to-purchase-device-slots.html) for more info."`